app/src/main/java/io/gnosis/safe/HeimdallApplication.kt

```diff
@@ -42,3 +49,26 @@ class HeimdallApplication : MultiDexApplication(), ComponentProvider {
         }
     }
 }
+
+private class CrashReportingTree : Timber.Tree() {
+
+    override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
+
+        if (priority == Log.VERBOSE || priority == Log.DEBUG) {
+            return
+        }
+        
+        t?.let {
+            if (priority == Log.ERROR) {
+                with(FirebaseCrashlytics.getInstance()) {
+                    setCustomKey(CRASHLYTICS_KEY_LOCALE, Locale.getDefault().language)
```

Comment:
Why only report the language? Why not language and region/country? en_US is different from en_GB or pt_PT/pt_BR. 
I'd go for: 
```
                    val locale = Locale.getDefault()
                    setCustomKey(CRASHLYTICS_KEY_LOCALE, "${locale.isO3Language}_${locale.isO3Country}")
```

Reply:
sure, why not